v0.5/mlperf.conf

```diff
@@ -9,27 +9,28 @@
 *.SingleStream.min_query_count = 1024
 
 *.MultiStream.target_qps = 1.0
-*.MultiStream.target_latency_percentile = 90
+*.MultiStream.target_latency_percentile = 99
 *.MultiStream.samples_per_query = 4
 *.MultiStream.max_async_queries = 1
 *.MultiStream.target_latency = 50
 *.MultiStream.min_duration = 60000
 *.MultiStream.min_query_count = 270336
-gnmt.MultiStream.min_query_count = 90112
 mobilenet.MultiStream.target_latency = 66
```

Comment:
Just realized this. Shouldn't this be `ssd-resnet34.MultiStream.target_latency = 66`?

Reply:
I copied it from here:
https://github.com/mlperf/inference_policies/blob/master/inference_rules.adoc#41-benchmarks

but I think you are right, the rules doc is wrong.

Reply:
https://github.com/mlperf/inference_policies/issues/77